### PR TITLE
Fix/HDH 111

### PR DESCRIPTION
**Pull request by rohanmrzan100**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ yes] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ yes] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,11 +6,11 @@
   "scripts": {
     "docusaurus": "docusaurus",
     "start": "docusaurus start --host 0.0.0.0",
-    "build": "export NODE_OPTIONS=--max-old-space-size=4096 && docusaurus build",
+    "build": "SET NODE_OPTIONS=--max-old-space-size=4096 && docusaurus build",
```
Comment (wei-harness) [inline package.json:9]: Please do not commit command for Windows, change back to "export"

**Review by wei-harness — CHANGES_REQUESTED** [review]
Please do not commit command ("SET") for Windows, change back to "export"

**Comment by wei-harness** [comment]
RSS preview: https://64a5887ba4f7e700729c590f--harness-developer.netlify.app/release-notes/rss.xml

Screen Shot 2023-07-05 at 8 03 53 PM

**Review by wei-harness — APPROVED** [review]
LGTM

**Comment by ravilach** [comment]
Awesome. Thank you @rohanmrzan100 we have been needing this w3c validation for a while!

**Review by ravilach — APPROVED** [review]

**Pull request merged**